### PR TITLE
Integrate IO

### PR DESCRIPTION
**Pull request by danieldortega77**
Integrate the components from the analysis page, Java servlet, and JavaScript. Error handling for attribute selection and languages needs to be added in a future PR.

### CODE REVIEW

**Comment by olu144** [comment]
Everything works well together!

**Comment by chrismohrii** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fstep40-2020%2Fpull%2F11) for more info**.

@googlebot I fixed it.

File: capstone/src/main/java/com/google/sps/servlets/CallPerspectiveServlet.java [hunk 1]
```diff
@@ -33,9 +33,9 @@
 import java.util.Arrays;
 
 /** Servlet that returns Perspective scoring. */
-@WebServlet("/call-perspective")
+@WebServlet("/call_perspective")
 public class CallPerspectiveServlet extends HttpServlet {
-  private static final String URL = "https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=API_KEY";
+  private static final String URL = "https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=NO";
```
Comment (chrismohrii) [inline capstone/src/main/java/com/google/sps/servlets/CallPerspectiveServlet.java:38]: Could we replace `NO` with `API_KEY`?

File: capstone/src/main/webapp/script.js [hunk 2]
```diff
@@ -37,9 +53,27 @@ function displayPerspectiveOutput(toxicityData) {
...
+          outputElement.appendChild(attributeElement);
+        }
+        if (document.getElementById('profanity').checked == true && key == "PROFANITY") {
+          attributeElement = createAnyElement('p', key + ": " + toxicityData.attributeScores[key].summaryScore.value);
+          outputElement.appendChild(attributeElement);
+        }
+        if (document.getElementById('threat').checked == true && key == "THREAT") {
+          attributeElement = createAnyElement('p', key + ": " + toxicityData.attributeScores[key].summaryScore.value);
+          outputElement.appendChild(attributeElement);
+        }
```
Comment (chrismohrii) [inline capstone/src/main/webapp/script.js:76]: Could we do this more efficiently in the servlet without requesting extra attributes? I would also want to discuss what exactly we want to show the user - there are many different things we could do

**Review by chrismohrii — CHANGES_REQUESTED** [review]

File: capstone/src/main/webapp/script.js (hunk 2 above)
Comment (danieldortega77) [reply capstone/src/main/webapp/script.js:76]: I agree - this could be handled by the servlet

**Comment by danieldortega77** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fstep40-2020%2Fpull%2F11) for more info**.

@googlebot I consent.

File: capstone/src/main/java/com/google/sps/servlets/CallPerspectiveServlet.java (hunk 1 above)
Comment (danieldortega77) [reply capstone/src/main/java/com/google/sps/servlets/CallPerspectiveServlet.java:38]: +1

**Comment by olu144** [comment]
@googlebot I consent

**Pull request closed without merging**